WordPressAuthenticator/Credentials/WordPressComCredentials.swift

```diff
@@ -19,17 +19,19 @@ public struct WordPressComCredentials: Equatable {
     /// The site address used during login
     ///
     public var siteURL: String
-
-    /// Designated initializer
+    
+    private let wpComURL = "https://wordpress.com"
```

Comment:
"https://wordpress.com" is used in multiple locations; maybe not in this PR, but I'm wondering if it makes sense to consolidate the constant in just one place?

Reply:
While I agree with the observation, it was my feeling that trying to address this as part of these PRs is a bit beyond scope.  I think consolidating these constants requires some analysis of its own that would risk causing scope creep in these PRs.

Reply:
Totally. I noticed this here but it needs not to be addressed in this PR.